nixos/modules/tasks/network-interfaces.nix

```diff
@@ -1087,7 +1087,24 @@ in
 
     virtualisation.vswitch = mkIf (cfg.vswitches != { }) { enable = true; };
 
-    services.udev.packages = mkIf (cfg.wlanInterfaces != {}) [
+    services.udev.packages =  [
+      (pkgs.writeTextFile rec {
+        name = "ipv6-privacy-extensions.rules";
+        destination = "/etc/udev/rules.d/98-${name}";
+        text = ''
+          # enable and prefer IPv6 privacy addresses by default
+          ACTION=="add", SUBSYSTEM=="net", RUN+="${pkgs.procps}/bin/sysctl net.ipv6.conf.%k.use_tempaddr=2"
```

Comment:
It fails if interface name contains `.` (which should be replaced with `/`)

Reply:
Sep 06 20:48:27 don1 systemd-udevd[4259]: Process '/nix/store/iysymmqj6c21fdqam8p9lljlly90fxsz-procps-3.3.15/bin/sysctl net.ipv6.conf.tinc.dmz.use_tempaddr=2' failed with exit code 255.